maro/event_buffer/event_list.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from .atom_event import AtomEvent
+from .cascade_event import CascadeEvent
+from .event_state import EventState
+from .maro_events import MaroEvents
+from .typings import Event, EventList, Union
+
+
+class EventLinkedList:
```

Comment:
the naming of this file is better to align with the defined class

Reply:
changed